src/main/java/se/sundsvall/parkingpermit/integration/camunda/mapper/CamundaMapper.java

```diff
@@ -0,0 +1,24 @@
+package se.sundsvall.parkingpermit.integration.camunda.mapper;
+
+import static se.sundsvall.parkingpermit.Constants.CAMUNDA_VARIABLE_CASE_NUMBER;
+
+import java.util.Map;
+
+import generated.se.sundsvall.camunda.StartProcessInstanceDto;
+import generated.se.sundsvall.camunda.VariableValueDto;
+
+public class CamundaMapper {
+	private CamundaMapper() {}
+
+	public static StartProcessInstanceDto toStartProcessInstanceDto(String caseNumber) {
+		return new StartProcessInstanceDto()
+			.businessKey(caseNumber)
+			.variables(Map.of(CAMUNDA_VARIABLE_CASE_NUMBER, toVariableValueDto(caseNumber)));
+	}
+
+	private static VariableValueDto toVariableValueDto(String caseNumber) {
+		return new VariableValueDto()
+			.type(Long.class.getSimpleName())
```

Comment:
Make a private static variable for this, instead of calling: Long.class.getSimpleName() every time?

`private static final String CASE_NUMBER_TYPE = Long.class.getSimpleName();`